build-scripts/cassandra-dtest-pytest.sh

```diff
@@ -86,8 +86,12 @@ fi
 
 SPLIT_TESTS=""
 if [ "x${DTEST_SPLIT_CHUNK}" != "x" ] ; then
-    ./run_dtests.py --cassandra-dir=$CASSANDRA_DIR ${DTEST_ARGS} --dtest-print-tests-only --dtest-print-tests-output=${WORKSPACE}/test_list.txt 2>&1 > ${WORKSPACE}/test_stdout.txt
-    SPLIT_TESTS=$(split -n r/${DTEST_SPLIT_CHUNK} ${WORKSPACE}/test_list.txt)
+    ./run_dtests.py --cassandra-dir=$CASSANDRA_DIR ${DTEST_ARGS} --dtest-print-tests-only --dtest-print-tests-output=${WORKSPACE}/reuseClusterTests.txt --pytest-options "-m reuse_cluster" 2>&1 > ${WORKSPACE}/test_stdout_reuse.txt
+    ./run_dtests.py --cassandra-dir=$CASSANDRA_DIR ${DTEST_ARGS} --dtest-print-tests-only --dtest-print-tests-output=${WORKSPACE}/renewClusterTests.txt --pytest-options "-m \"not reuse_cluster\"" 2>&1 > ${WORKSPACE}/test_stdout_renew.txt
+    CHUNK=`echo ${DTEST_SPLIT_CHUNK}|cut -d "/" -f 1`
+    SPLITS=`echo ${DTEST_SPLIT_CHUNK}|cut -d "/" -f 2`
+    ./splitter.sh ${CHUNK} ${SPLITS} ${WORKSPACE}/reuseClusterTests.txt ${WORKSPACE}/renewClusterTests.txt ${WORKSPACE}/test_list.txt
```

Comment:
where is the `splitter.sh` file?

Reply:
In the dtests repro bc it needs to be used both in circle and jenkins https://github.com/apache/cassandra-dtest/pull/160/files#diff-d881c74783d321878b9e65e0027c2d5930091daa970ea1aed5c5120464af553d